Makefile

```diff
@@ -314,30 +314,39 @@ upload-rpms: rpmbuild/RPMS/ rpmbuild/RPMS/SHA256SUMS rpmbuild/RPMS/SHA256SUMS.as
 #
 #	copr build
 #
-copr: upload-srpms
+copr: upload-srpms ## build in copr
 	./misc/copr-build.py https://$(SERVER)/$(REMOTE_PATH)/SRPMS/$(SRPM_BASE)
 
 #
 #	deb build
 #
 
-deb:
+deb: ## build debian package
 	./misc/gen-deb-changelog-from-git.sh
 	dpkg-buildpackage
 	# especially when building in Docker container, pull build artifact in project directory.
 	cp ../mgmt_*_amd64.deb ./
 	# cleanup
 	rm -rf debian/mgmt/
 
-build_container:
+build_container: ## builds the container
 	docker build -t purpleidea/mgmt-build -f docker/Dockerfile.build .
 	docker run -td --name mgmt-build purpleidea/mgmt-build
 	docker cp mgmt-build:/root/gopath/src/github.com/purpleidea/mgmt/mgmt .
 	docker build -t purpleidea/mgmt -f docker/Dockerfile.static .
 	docker rm mgmt-build || true
 
-clean_container:
+clean_container: ## removes the container
 	docker rmi purpleidea/mgmt-build
 	docker rmi purpleidea/mgmt
 
+help: ## Show this help screen
```

Comment:
nit: this is the only help message which is capitalized, want to make it uniform and lower case it please?